unity-client/Assets/Builder/Scripts/DCLBuilderObjectSelector.cs

```diff
@@ -63,21 +73,29 @@ private void OnDisable()
         {
             isGameObjectActive = false;
             DCLBuilderInput.OnMouseDown -= OnMouseDown;
-            DCLBuilderInput.OnMouseDrag -= OnMouseDrag;
             DCLBuilderInput.OnMouseUp -= OnMouseUp;
             DCLBuilderBridge.OnResetObject -= OnResetObject;
+            DCLBuilderBridge.OnEntityAdded -= OnEntityAdded;
             DCLBuilderBridge.OnEntityRemoved -= OnEntityRemoved;
-            DCLBuilderBridge.OnSetGridResolution -= OnSetGridResolution;
             DCLBuilderBridge.OnSceneChanged -= OnSceneChanged;
             DCLBuilderBridge.OnBuilderSelectEntity -= OnBuilderSelectEntity;
-            DCLBuilderBridge.OnBuilderDeselectEntity -= OnBuilderDeselectEntity;
+            DCLBuilderGizmoManager.OnGizmoTransformObject -= OnGizmoTransform;
+            DCLBuilderGizmoManager.OnGizmoTransformObjectEnd -= OnGizmoTransformEnded;
+            DCLBuilderObjectDragger.OnDraggingObject -= OnObjectsDrag;
+            DCLBuilderObjectDragger.OnDraggingObjectEnd -= OnObjectsDragEnd;
         }
 
         private void Update()
         {
-            if (!gizmosManager.isTransformingObject)
+            if (isDirty)
```

Comment:
We can invert the if to save indentation.
```suggestion
            if (!isDirty)
                return;
```